docs/pages/l2_dsCompositeIn_test_specification.md

```diff
@@ -0,0 +1,125 @@
+### Test 1
+
+|Title|Details|
+|--|--|
+|Function Name|`test_l2_dsCompositeIn_GetNumberOfInputs`|
+|Description|Verify that the function returns the expected COMPOSITE Input ports.|
+|Test Group|Module : 02|
+|Test Case ID|001|
+|Priority|High|
+
+**Pre-Conditions :**
+None
+
+**Dependencies :**
+None
+
+**User Interaction :**
+If user chose to run the test in interactive mode, then the test case has to be selected via console.
+
+#### Test Procedure :
+
+| Variation / Steps | Description | Test Data | Expected Result | Notes|
+| -- | --------- | ---------- | -------------- | ----- |
+| 01 | Initialize the COMPOSITE Input ports using dsCompositeInInit | None | dsERR_NONE | Should be successful |
+| 02 | Get the number of COMPOSITE Input ports using dsCompositeInGetNumberOfInputs | numInputs = valid buffer | dsERR_NONE | Should be successful |
+| 03 | Verify the returned number of COMPOSITE Input ports | numInputs = valid buffer | numInputs >= 0 | Should be successful |
```

Comment:
Verification should be done with the value read from the config file.  The above statement doesn't say with what should the numInputs should be validated.

Reply:
to be addressed with profile support upgrade

Reply:
Done